openedx/features/learner_profile/static/learner_profile/js/learner_profile_factory.js

```diff
@@ -131,7 +131,17 @@
                     options: options.language_options,
                     helpMessage: '',
                     persistChanges: true
+                }),
+
+                new FieldsView.ReadonlyFieldView({
```

Comment:
Thanks @andy-armstrong.  Yes.  Just making sure we were on the same page.  It may or may not make sense for me to do this before @HarryRein's story lands, but Diana and I can work that out next week.  Thanks!